docs/pages/7_0_0_release_notes.md

```diff
@@ -17,7 +17,55 @@ This is a {{ site.pmd.release_type }} release.
 
 {% tocmaker is_release_notes_processor %}
 
-### New and noteworthy
+### 🚀 New and noteworthy
+
+#### Release Candidates
+
+PMD 7.0.0 is finally almost ready. In order to ease the upgrade from PMD 6 to 7, we are going to ship a couple of
+release candidates. These are official pre-release versions available on GitHub and Maven Central and can be used
+as usual (e.g. as a dependency).
+
+Using the release candidate is also a good opportunity to provide feedback via our
+[issue tracker](https://github.com/pmd/pmd/issues/new/choose).
+
+#### New official logo
+
+Many of you probably have already seen the new logo, but now it's time to actually ship it. The new logo
+was long ago decided (see [#1663](https://github.com/pmd/pmd/issues/1663)).
+We decided it's time to have a modernized logo and get rid of the gun. This allows to include
+the logo anywhere without offense.
+
+
+![New PMD Logo](images/logo/pmd-logo-300px.png)
+
+#### Revamped Java module
+
+The Java grammar has been refactored substantially in order to make it easier to maintain and more correct
+regarding the Java Language Specification. Changing the grammar entails a changed AST and therefore changed
+rules. The PMD built-in rules have all been upgraded and many bugs have been fixed on the way.
+Unfortunately, if you are using custom rules, you will most probably need to accommodate these changes yourself.
+
+The type resolution framework has been rewritten from scratch and should now cover the entire Java spec correctly.
+The same is true for the symbol table.
+PMD 6 on the other hand has always had problems with advanced type inference, e.g. with lambdas and call chains. 
+Since it was built on the core reflection API, it also was prone to linkage errors and classloader leaks for instance.
+PMD 7 does not need to load classes, and does not have these problems.
+
+The AST exposes much more semantic information now. For instance, you can jump from a method call to 
+the declaration of the method being called, or from a field access to the field declaration. These
+improvements allow interesting rules to be written that need precise knowledge of the types
+in the program, for instance to detect unnecessary boxing or unnecessary explicit type arguments (TODO link those rules).
+These are just a small preview of the new rules we will be adding in the PMD 7 release cycle.
+
+Some first results of the Java AST changes are for now documented in the Wiki:
+[Java clean changes](https://github.com/pmd/pmd/wiki/Java_clean_changes).
```

Comment:
I think for rc1, we can keep the link to the wiki page. We have #4294 to address this.

Reply:
I started to move it into the detailed release notes page and added a couple of TODOs.